.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,51 @@
+name: Release metrics adapter chart
+on:
+  workflow_dispatch:
```

Comment:
in case of a manual release of the chart only, is there going to be a GH release specifically for that?

Reply:
Yep, chart releaser creates it automatically. But we changed the logic of this and now it releases it when it touches main so we should not care for this anymore.